waproto/src/whatsapp.proto

```
@@ -5455,6 +6161,17 @@ message ThreadID {
   }
 }
 
+message UnCountedAssociatedMessageList {
+  repeated WebMessageInfo messages = 1;
+  optional MessageKey parentMessage = 2;
+  optional MessageAssociation.AssociationType associationType = 3;
+}
+
+message UnCountedAssociatedMessageListWithMessageBytes {
+  repeated WebMessageInfoWithMessageBytes messages = 1;
+  optional MessageKey parentMessage = 2;
```

Buf (1.70.0)

[error] Field name "parentMessage" should be lower_snake_case, such as "parent_message".

(FIELD_LOWER_SNAKE_CASE)